src/test/java/org/jenkinsci/plugins/docker/workflow/DockerUtilsTest.java

```diff
@@ -23,14 +23,17 @@
  */
 package org.jenkinsci.plugins.docker.workflow;
 
+import hudson.EnvVars;
 import hudson.FilePath;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
 import org.hamcrest.collection.IsCollectionWithSize;
 import org.hamcrest.core.IsCollectionContaining;
```

Comment:
Imports to facilitate changes to test
```suggestion
 import org.hamcrest.core.IsMapContaining;
```